lib/nga.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
```

Comment:
More descriptive file name? It's a bit cryptic.

Reply:
It feels like this and the next comment can be resolved by renaming the script `tiff_to_pyramid.bash`.  Let's do that.